### ISSUE TITLE
Add support for armhf 32 bits architecture

### ISSUE THREAD

**Issue by oijkn** [LibrePhotos/librephotos-docker#32]
I use a Raspberry Pi 4 with a 32 bits architecture and I would like to use LibrePhotos. How can I build the different docker to be compatible with my hardware, please?

BR.

**Comment by wrapper** [comment]
+1

**Comment by quartztester** [comment]
Would this issue not be more appropriate in the dedicated LibrePhotosOS? 

https://github.com/guysoft/LibrePhotosOS/issues
Requirements
"Raspberrypi 3 and above, needs a 64bit capable pi"

**Comment by derneuere** [comment]
No, we need to provide actual armhf32 images for LibrePhotosOS first. Right now, we only build arm64 and amd64.

I think the pipeline mostly supports this, but we probably have to look for a pytorch build in armhf32 or implement deactivating AI algorithms on low power devices.

**Comment by Gottarocket** [comment]
waiting for the armv7 support

**Comment by derneuere** [comment]
Will not implement :)
